src/main/java/it/gov/pagopa/pu/classification/repository/AssessmentsDetailRepository.java

```diff
@@ -21,6 +21,9 @@ public interface AssessmentsDetailRepository extends JpaRepository<AssessmentsDe
   @RestResource(exported = false)
   AssessmentsDetail findByDebtPositionTypeOrgCodeAndIuvAndIudAndOfficeCodeAndSectionCodeAndAssessmentCode(String debtPositionTypeOrgCode, String iuv, String iud, String officeCode, String sectionCode, String assessmentCode);
 
+  @RestResource(exported = false)
+  void deleteAllByDebtPositionTypeOrgCodeAndIuvAndIud(String debtPositionTypeOrgCode, String iuv, String iud);
```

Comment:
why `debtPositionTypeOrgCode`?
```suggestion
  void deleteAllByOrganizationIdAndIuvAndIud(String debtPositionTypeOrgCode, String iuv, String iud);
```